README.md

```diff
@@ -18,14 +18,30 @@ spatial local launch
 
 (Replacing `windows` with `macos` on macOS, or `linux` on Linux).
 
-This will launch SpatialOS locally with a single C# "physics" worker that updates the position of
-a single entity. You can then connect either one of the two C client workers (one implemented using
-"direct" serialization, the other implemented using "vtable" serialization). These workers can be
+This will launch SpatialOS locally with a single C++ "physics" worker that updates the position of
+a single entity. You may also see a 2nd entity called "physics-worker" created. This entity
+represents the managed worker itself.
+
+Note: If you run `spatial worker build` without a `--target` parameter (or with the wrong target
+parameter), then the CMake cache for each worker (`workers/<worker>/cmake_build`) may end up in
+a corrupt state. To recover, just run `spatial worker clean` to delete the CMake caches.
+
+Now, you can connect either one of the two C client workers (one implemented using "direct"
+serialization, the other implemented using "vtable" serialization). These workers can be
 launched with the following commands:
 
 * Client (direct): `spatial local worker launch client_direct local`
 * Client (vtable): `spatial local worker launch client_vtable local`
 
+No output will be written, but you should see a 2nd worker connect in the inspector. To view the
+printf logging from the C workers, launch the executable directly. For example, on Windows:
+
+```
+./workers/c_client_direct/cmake_build/Debug/ClientDirect.exe localhost 7777 ClientDirect1
+```
+
+(note that the 3rd parameter should be a different string every time you run a worker).
```

Comment:
Nit: make it more explicit that the string should be unique for a single deployment. A deployment restart allows for string reuse.

Reply:
I decided to fix this by making the clients generate a random suffix alongside the ID.